health-monitors/system-services-monitor/system_services_monitor/checkers/service_check.py

```
@@ -0,0 +1,273 @@
+        try:
+            result = self._run_host_cmd([
+                "systemctl",
+                "show",
+                service_name,
+                "--property=NRestarts",
+            ])
+            if result.returncode == 0 and result.stdout.strip():
+                _, _, val = result.stdout.strip().partition("=")
+                return int(val)
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)